pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -1050,10 +1073,11 @@ func NewPodNominator(podLister listersv1.PodLister) framework.PodNominator {
 	}
 }
 
-// MakeNextPodFunc returns a function to retrieve the next pod from a given
-// scheduling queue
-func MakeNextPodFunc(queue SchedulingQueue) func() *framework.QueuedPodInfo {
-	return func() *framework.QueuedPodInfo {
+// MakeNextPodFunctions returns a function to retrieve the next pod from a given
```

Comment:
MakeNextPodFuncs returns two functions to ...

Reply:
Changed.